home/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
 # Create your models here.
+
+class UserInfo(models.Model):
```

Comment:
@shahriaralamsakib  Keep the model name `User` and it should be inherited from Django's abstructbase user model. Get help from internet what is abstrucrbaseuser model

Reply:
There should be 2 types of user,
1. Cusromer
2. Admin
3. Super User

You can handle the types using a textchoice field